configure

```diff
@@ -656,16 +656,26 @@ write_action_env_to_bazelrc "TF_CUDNN_VERSION" "$TF_CUDNN_VERSION"
 
 # Configure the compute capabilities that TensorFlow builds for.
 # Since Cuda toolkit is not backward-compatible, this is not guaranteed to work.
+function get_native_cuda_compute_capabilities {
```

Comment:
Yes, you are right. And I think this is done automatically (Not tested on Windows yet. I don't have a Windows machine available now for me to test).

In case of errors, for example if on windows, or the samples are not installed with cuda, etc. The function `get_native_cuda_compute_capabilities` will exit without outputting anything (and the configure script will keep running). In this case, `native_cuda_compute_capabilities` will be empty and
```bash
default_cuda_compute_capabilities=${native_cuda_compute_capabilities:-"3.5,5.2"}
```

will automatically gives `3.5,5.2`